engine/src/test/java/io/zeebe/engine/processor/RandomDurationTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.processor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import org.junit.Test;
+
+public class RandomDurationTest {
```

Comment:
Just a thought: as we deal with random values, does it make sense here to test that? e.g. most of these tests would pass if `getDuration` returned a fixed value. 

At the same time, the logic is so simple not sure if it's worth it :man_shrugging: :

Reply:
I changed it to minutes based where it might have more logic now and it tests that it is between this given values but yes it is quite simple